### DISCUSSION TITLE
different from the 4.0 binaries in any useful way?

### DISCUSSION THREAD

**Discussion by alanbork** [OpenSalamander/salamander#55] [General]
is this version worth building just for general use, or is 5.0 just the same as 4.0, but open source(d)?

**Comment by melloware** [comment]
@alanbork i am not positive but i don't think the current code is much different than 4.0...YET.  The first step is translating all the code comments to English.

**Comment by janrysavy** [comment]
Only difference is patch for [CSV crash](https://forum.altap.cz/viewtopic.php?t=42577) and some deprecated plugins missing.
